src/crewai/cli/run_crew.py

```diff
@@ -26,18 +36,54 @@ def run_crew() -> None:
             fg="red",
         )
 
+    # Determine crew type
+    is_flow = pyproject_data.get("tool", {}).get("crewai", {}).get("type") == "flow"
+    crew_type = CrewType.FLOW if is_flow else CrewType.STANDARD
```

Comment:
should this be a util method ? I know we aren't using this elsewhere now but would be great inside project/utils 

```python
def get_project_type(): -> CrewType
```